6-AdvancedScenarios/4-hybrid-SPA/App/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "hybrid-spa-app",
```

Comment:
```suggestion
  "name": "hybrid-spa-app",
```

let's follow the naming convention used in the rest of the samples

Reply:
Done!